### PR TITLE
143345561 timesince refactor

### PR DESCRIPTION
**Pull request by Laszlo-JFLMTCO**
@jdconrad89 @mollybrown @drod1000 
This PR covers [ticket ID 143345561](https://www.pivotaltracker.com/story/show/143345561).

Please note that there have been no tests previously to confirm the `created_at` detail on the `dashboard` page. This PR includes commented out evaluations to confirm project created_at text and project name on dashboard page. Implementation is functional, but another test setup issue that fails... This is part of a growing set of problems with several test cases failing, while the implementation is functional (confirmed through manual testing). We will need to discuss that as a team.

FYI @neight-allen

### CODE REVIEW

**Comment by drod1000** [comment]
@Laszlo-JFLMTCO Awesome! Do you think you could include a screenshot since we haven't been able to write a test that passes?

File: spec/features/portfolios/user_can_create_portfolio_spec.rb [hunk 1]
```diff
@@ -47,7 +47,7 @@
       fill_in 'linkedin', with: "www.test.com"
 
       click_on 'Save as Draft'
-      sleep(1)
+      sleep(5)
```
Comment (mollybrown) [inline spec/features/portfolios/user_can_create_portfolio_spec.rb:50]: Just out of curiosity, why increase the sleep amount?

**Comment by mollybrown** [comment]
Good call on using moment.js to handle the time math instead. I agree that the failing tests are an issue that need to be addressed, esp. since this project will be handed off again in a few weeks. I think we are going to run out of time on fixing the tests, so we will need to find a way to communicate what is working/not working clearly to the next group.

File: spec/features/portfolios/user_can_create_portfolio_spec.rb (hunk 1 above)
Comment (Laszlo-JFLMTCO) [reply spec/features/portfolios/user_can_create_portfolio_spec.rb:50]: @mollybrown Tried many different things trying to get Capybara to recognize components on the page. I really hope there is a better solution, but so far any better engineered direction I tried I have hit a roadblock.

**Comment by Laszlo-JFLMTCO** [comment]
@drod1000 I am adding screenshot showing `created_at` information right after a new project has been created. This is with `moment.js` running.
screen shot 2017-04-12 at 3 06 31 am
For reference, here is a link to the relevant part of the `moment.js` library documentation, which has a nice table showing the definition of what text to be returned based on the time between `now` and a project's `created_at`: https://momentjs.com/docs/#/displaying/fromnow/
And a screenshot of the table from that link:  
image

**Comment by Laszlo-JFLMTCO** [comment]
@mollybrown I opened a ticket to document the test related concerns, it is in icebox: [ticket ID 143533323](https://www.pivotaltracker.com/story/show/143533323) Let's review as a team to confirm if it is sufficient to raise the concern for future follow up or update it or open another one.

**Comment by drod1000** [comment]
@Laszlo-JFLMTCO Awesome! This is a lot cleaner. I'm on board with merging this one and creating a ticket for the tests since it seems to be an issue that we keep running into.

**Pull request merged**